commands/Tools/artist.js

```diff
@@ -0,0 +1,36 @@
+// Requires getSpotifyToken task
+const { Command } = require('klasa');
+const fetch = require('node-fetch');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			cooldown: 10,
+			description: 'Searches Spotify for an artist, and returns an embed.',
+			usage: '<query:str{1,100}>'
+		});
+	}
+
+	async run(msg, [query]) {
+		if (!this.client._spotifyToken) {
```

Comment:
I'm not completely convinced with this. This should be externalized as a constant like all other pieces that use an API key.

Reply:
its not a constant value though, it changes every hour?

Reply:
Oh right, you update it in tasks/getSpotifyToken. You should make a large JSDoc specifying that requirement.